tests/functional/r/regression_02/regression_5801.py

```diff
@@ -0,0 +1,4 @@
+# pylint: disable=missing-module-docstring, missing-docstring
```

Comment:
Please add a link to the issue in a docstring. Issue trackers come and go and issue numbers do as well 😄

Reply:
👍 Upgraded the suggestion in the other comment